app/services/conversation_service/creator.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module ConversationService
+  class Creator
+    def initialize(sender:, receivers:, topic:)
+      @sender = sender
+      @receivers = receivers
+      @topic = topic
+      @conversations = []
+    end
+
+    def call
+      @receivers.each do |receiver|
```

Comment:
A few comments here:

1. WDYT about creating some reader methods instead of accessing the instance variables directly?
2. Instead of doing a `.each` and add elements to an Array, we should use `map`
3. Instead of calling `create` multiple times, we can just provide an Array of Hashes and trigger all the inserts at once
```ruby
# Creating an Array of new objects using a block, where the block is executed for each object:
User.create([{ :first_name => 'Jamie' }, { :first_name => 'Jeremy' }]) do |u|
  u.is_admin = false
end
```
https://apidock.com/rails/ActiveRecord/Base/create/class